### ISSUE TITLE
[JOSE] Jose Paper

### ISSUE THREAD

**Issue by BenjMy** [cgre-aachen/gemgis_data#19]
xref https://github.com/openjournals/jose-reviews/issues/185

Overall it is good start, but reads a bit choppy. I found the article very ressource exploitation (oil, mining, ...) target-oriented while I think this tools has many other disciplines to reach.
 
   - _Statement of needs_
   	
     - To me for instance the tools developed can be also associated with environmental risks management (water scarcity, landslides, ...). I am personnaly interested to this package for Catchement hydrologeology modelling. Authors should broaden this idea in the statement of needs.
   
     - I like the proof of concept and that authors already used the software for teaching to RWTH Aachen. Could it be possible to add (in the "Experience of use in teaching and learning situations" section): 
     	- a link to the degree where it has been tought;
     	- the number of hours dedicated to the course (and how it has been distributed over all the notebooks);
     	- the ratio between the number of instructors and number of students.
     - "Even from seismic data" is to me again to me to narrow. Why other geophysical approaches able to infer geological structures (TEM, gravity, ...) are not considered? 
      
      
   - _Ressources_:    
     - I suggest to change the order of ressources and place "GemGIS" first and then add "GemPy" indicating that it is an optionnal dependency. 
     - "Tutorial repository" link and "GemGIS link" are redundant. Choose one or another or change the first to "https://gemgis.readthedocs.io/en/latest/getting_started/tutorial/index.html"

   - _References_: 
   Seems that references are broken in the paper compilation.

**Comment by jwagemann** [comment]
Hi. I am adding additional comments to each issue @BenjMy already started.

* References in the paper seem not to be properly rendered
* Statement of Need needs refinement and need to be clearer
  * What does ‘paper-based analog approach’ mean?
  * I think the scope and target audience can be a bit broader as I see value in the tutorial being of interest for everyone who is interested in using the two open-source packages GemGIS and GemPy
* Make sure to spell out acronyms - e.g. what are ‘AR-sandboxes’?
* you claim that the material can easily be used by other instructors. However, I doubt that this is the case at the moment. I would suggest to make the notebooks more educational and self-explanatory
* From the paper, it is currently hard to follow and grasp the structure and organisation of the course - this is also due to the fact that the course itself can be restructured. See my comment [here](https://github.com/cgre-aachen/gemgis_data/issues/18)

**Comment by AlexanderJuestel** [comment]
But here also a list to track my progress:

   - _Statement of needs_
   	

- [ ] To me for instance the tools developed can be also associated with environmental risks management (water scarcity, landslides, ...). I am personnaly interested to this package for Catchement hydrologeology modelling. Authors should broaden this idea in the statement of needs. --> **This also relates to #23**

   

- [ ] I like the proof of concept and that authors already used the software for teaching to RWTH Aachen. Could it be possible to add (in the "Experience of use in teaching and learning situations" section): 

     	- a link to the degree where it has been tought;
     	- the number of hours dedicated to the course (and how it has been distributed over all the notebooks);
     	- the ratio between the number of instructors and number of students.

- [ ]  "Even from seismic data" is to me again to me to narrow. Why other geophysical approaches able to infer geological structures (TEM, gravity, ...) are not considered? 

      
      
   - _Ressources_:    

- [x] I suggest to change the order of ressources and place "GemGIS" first and then add "GemPy" indicating that it is an optionnal dependency. 

 

- [x]  "Tutorial repository" link and "GemGIS link" are redundant. Choose one or another or change the first to "https://gemgis.readthedocs.io/en/latest/getting_started/tutorial/index.html"

   - _References_: 
   

- [ ]  Seems that references are broken in the paper compilation. --> **The references compile using Github Actions in the gemgis_data repo**

**Comment by BenjMy** [comment]
Hi @AlexanderJuestel,

Thanks for taking into consideration all my comments. Nothing the add here from my side.

**Comment by AlexanderJuestel** [comment]
@BenjMy, thanks for that! I think I forgot to go over @jwagemann comments here. I will add them to my list above and will go through them right away

**Comment by AlexanderJuestel** [comment]
@jwagemann I tried to add some information to the manuscript based on your comments. Please feel free to review the changes and let me know if I can close this issue here
